src/map/attackround.cpp

```diff
@@ -301,24 +303,29 @@ void CAttackRound::CreateAttacks(CItemWeapon* PWeapon, PHYSICAL_ATTACK_DIRECTION
     // Quad/Triple/Double Attack
     else if (xirand::GetRandomNumber(100) < quadAttack)
     {
-        AddAttackSwing(PHYSICAL_ATTACK_TYPE::QUAD, direction, 3);
+        AddAttackSwing(PHYSICAL_ATTACK_TYPE::QUAD, direction, 4);
```

Comment:
They aren't adding the "first swing" anymore near the bottom, I think this means a call of `AddAttackSwing(PHYSICAL_ATTACK_TYPE::QUAD, direction, 4);` results in 4 hits of QUAD. It currently seems to only give 3 hits of QUAD

Reply:
Could you add back the Quad attack setting `multiHitOccurred` to true? This way, DA/TA/QA have the same pattern of adding the same amount of hits as their name. It could be confusing otherwise. Thanks!

Reply:
Restored multiHitOccurred to QA
Restored QA AddAttackSwing value to 4